hoomd/md/ForceComposite.h

```diff
@@ -73,7 +73,50 @@ class PYBIND11_EXPORT ForceComposite : public MolecularForceCompute
          */
         virtual void validateRigidBodies(bool create=false);
 
-    protected:
+        
+        /// Construct from a Python dictionary
+        void setBody(std::string typ, pybind11::object v)
+            {   
+            pybind11::list types = v["types"];
+            pybind11::list positions = v["positions"];
+            int N = (unsigned int)len(positions);
+
+            // extract the positions from the python list
+            std::vector<vec3<OverlapReal>> pos_vector;
```

Comment:
```suggestion
            std::vector<vec3<Scalar>> pos_vector;
```
`OverlapReal` is an HPMC specific data type. You want `Scalar` here.